sympy/matrices/matrices.py

```diff
@@ -2139,6 +2139,9 @@ def LUdecomposition_Simple(self, iszerofunc=_iszero, simpfunc=None,
     def LUdecompositionFF(self):
         return _LUdecompositionFF(self)
 
+    def SVD(self):
```

Comment:
```suggestion
    def singular_value_decomposition(self):
```
Maybe I'd rename like

Reply:
I've renamed the function in the latest commit and also added more tests with symbolic matrices